flutter/realm_flutter/tests/pubspec.yaml

```diff
@@ -21,19 +21,14 @@ dependencies:
     path: ../
 
 dev_dependencies:
-  flutter_driver:
-    sdk: flutter
   flutter_test:
     sdk: flutter  
   flutter_lints: ^3.0.1
-  build_runner: ^2.1.2
   test: any
   timezone: ^0.9.0
+  integration_test:
+    sdk: flutter
 
 flutter:
-  uses-material-design: true
   assets:
-    - data/realm_files/old-format.realm
-
-dependency_overrides:
-  test_api: ^0.6.1
+    - test/data/realm_files/old-format.realm
```

Comment:
If we're defining the test data files here, can we get rid of the symlink and instead use `../../.../../test/data/realm_files/old-format.realm`?

Reply:
We need the asset path and the actual path to be the same currently, for the file to be copied correctly. But I guess we could do something clever.

Reply:
I suggest leaving the data symlink for now.